### ISSUE TITLE
SsrEnv TS error when using the isolated Modules flag

### ISSUE THREAD

**Issue by yoroshikun** [brillout/vite-plugin-ssr#81]
Current export of SsrEnv does not take into consideration it is only a type as it must be considered when --isolatedModules flag is provided
```
node_modules/vite-plugin-ssr/ssrEnv.node.ts:5:10 - error TS1205: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.

5 export { SsrEnv }
           ~~~~~~


Found 1 error.
```

**Comment by brillout** [comment]
Curious: why do you use `--isolatedModules`? Thanks for the PR

**Comment by brillout** [comment]
Released in `0.1.0-beta.48`.
